README.md

```diff
@@ -1762,6 +1763,80 @@ const starkinfra = require('starkinfra');
 })();
 ```
 
+### Preview CreditNotes
+You can preview a Credit Note before creation of the CCB contract:
```

Comment:
You can create a Credit Note Previews to preview a CCB contract
based on a specific table type: